lib/main.dart

```diff
@@ -28,10 +29,11 @@ class MyApp extends StatelessWidget {
           canvasColor: canvasTheme,
           fontFamily: 'Lato',
         ),
-        home: ProductsOverviewScreen(),
+        initialRoute: ProductsOverviewScreen.routeName,
         routes: {
           ProductDetailScreen.routeName: (ctx) => ProductDetailScreen(),
           CartScreen.routeName: (ctx) => CartScreen(),
+          OrdersScreen.routeName: (ctx) => OrdersScreen(),
         },
```

Comment:
you might want to using `onGenerateRoute` just in case you will be passing arguments. please see these references

https://github.com/Nei1eveN/To-Do-App/blob/master/lib/main.dart#L26
https://github.com/Nei1eveN/To-Do-App/blob/master/lib/utils/app_router.dart